tools/accuracy_checker/accuracy_checker/annotation_converters/electricity_time_series_forecasting.py

```diff
@@ -0,0 +1,428 @@
+            {InputTypes.ID, InputTypes.TIME})
+
+        # Transform real inputs per entity
+        df_list = []
+        for identifier, sliced in df.groupby(id_col):
+
+          # Filter out any trajectories that are too short
+          if len(sliced) >= self._time_steps:
+            sliced_copy = sliced.copy()
+            sliced_copy[real_inputs] = self._real_scalers[identifier].transform(
+                sliced_copy[real_inputs].values)
+            df_list.append(sliced_copy)
+
+        output = pd.concat(df_list, axis=0)
+
+        # Format categorical inputs
+        for col in categorical_inputs:
+            string_df = df[col].apply(str)
+            output[col] = self._cat_scalers[col].transform(string_df)
+
+        return output
+
+    # Default params
+    def get_fixed_params(self):
+        """Returns fixed model parameters for experiments."""
+
+        fixed_params = {
+            'total_time_steps': 8 * 24,
+            'num_encoder_steps': 7 * 24,
+            'num_epochs': 100,
+            'early_stopping_patience': 5,
+            'multiprocessing_workers': 5
+        }
+
+        return fixed_params
+
+    def get_column_definition(self):
+        """"Returns formatted column definition in order expected by the TFT."""
+
+        column_definition = self._column_definition
+
+        # Sanity checks first.
+        # Ensure only one ID and time column exist
+        def _check_single_column(input_type):
+
+            length = len([tup for tup in column_definition if tup[2] == input_type])
+
+            if length != 1:
+                raise ValueError('Illegal number of inputs ({}) of type {}'.format(
+                    length, input_type))
+
+        _check_single_column(InputTypes.ID)
+        _check_single_column(InputTypes.TIME)
+
+        identifier = [tup for tup in column_definition if tup[2] == InputTypes.ID]
+        time = [tup for tup in column_definition if tup[2] == InputTypes.TIME]
+        real_inputs = [
+            tup for tup in column_definition if tup[1] == DataTypes.REAL_VALUED and
+            tup[2] not in {InputTypes.ID, InputTypes.TIME}
+        ]
+        categorical_inputs = [
+            tup for tup in column_definition if tup[1] == DataTypes.CATEGORICAL and
+            tup[2] not in {InputTypes.ID, InputTypes.TIME}
+        ]
+
+        return identifier + time + real_inputs + categorical_inputs
+
+
+class ElectricityTimeSeriesForecastingConverter(BaseFormatConverter):
+    """ Annotation converter for electricity dataset.
+    """
+    URL = 'https://archive.ics.uci.edu/ml/machine-learning-databases/00321/LD2011_2014.txt.zip'
+
+    __provider__ = "electricity"
+    annotation_types = (ElectricityTimeSeriesForecastingAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            "data_path": PathField(description="Path to dataset file in .csv format."),
```

Comment:
it will not be observable for concatenation with prefix. Please name this parameter with file suffix.

Reply:
done